src/BowlingGame.js

```diff
@@ -0,0 +1,66 @@
+var BowlingGame = function () {
+  this.frames = [];
+  this.totalScore = 0;
+  this._gameSetup();
+};
+
+BowlingGame.prototype.totalScoreBonuses = function () {
+  this._frameScoreBonuses();
+  var allBonuses = this.frames.reduce(function (totalBonus, frameBonus) {
```

Comment:
This line could be `return this.frames...` to avoid having to create a new var that is returned.